webui/src/app/stores/environments-store.service.ts

```diff
@@ -3,6 +3,7 @@ import { createFeatureSelector, createSelector, MemoizedSelector, Store } from '
 
 import { ActionWithPayload } from './action-with-payload';
 import { EnvironmentBean } from '../models/commons/applications-bean';
+import { Observable } from 'rxjs/internal/Observable';
```

Comment:
Sounds very weird to rely on someone else' internals! Auto-import typo for `rxjs/Observable`?